q2_types/per_sample_sequences/tests/test_transformer.py

```diff
@@ -102,5 +119,508 @@ def test_casava_one_eight_single_lane_per_sample_dirfmt_to_slpspefdf(self):
             self.assertEqual(act, exp)
 
 
+class TestFastqManifestTransformers(TestPluginBase):
+    package = "q2_types.per_sample_sequences.tests"
+
+    def test_single_end_fastq_manifest_phred33_to_slpssefdf(self):
+        format_ = SingleEndFastqManifestPhred33
+        transformer = self.get_transformer(
+            format_,
+            SingleLanePerSampleSingleEndFastqDirFmt)
+
+        with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
use `self.temp_dir` (derived from `TestPluginBase`) instead of all of these `TemporaryDirectory`s. `temp_dir` will have a reasonable name in case something goes wrong, making cleanup easier on the user.